src/OrganisationRegistry.UI/app/system/terminated-in-kbo/components/filter/filter.template.html

```diff
@@ -0,0 +1,42 @@
+<form class="form" [formGroup]="form" (ngSubmit)="filterForm(form.value)" autocomplete="off">
```

Comment:
Is this file unfinished?

Reply:
No, I left it there as a reminder we can expand on this later. Business requirements are still unclear here.